.gitignore

```diff
@@ -1,6 +1,8 @@
 node_modules
 build
+coverage
 gutenberg.pot
+.idea
```

Comment:
IDE specific entries in `.gitignore` should not be in the repo, users should manage these locally in `~/.gitignore_global`

See also https://core.trac.wordpress.org/ticket/34345